Script.js

```diff
@@ -203,3 +203,18 @@ contactForm.addEventListener('submit', (event) => {
     event.preventDefault();
   }
 });
+
+contactForm.addEventListener('input', () => {
+  const formInfo = {
+    name: document.querySelector('.contact_name').value,
+    email: document.querySelector('.email_address').value,
+    message: document.querySelector('.contact_message').value,
+  };
+
+  localStorage.setItem('contactForm', JSON.stringify(formInfo));
+});
+
+const formObj = JSON.parse(localStorage.getItem('contactForm'));
+document.querySelector('.contact_name').value = formObj.name;
+document.querySelector('.email_address').value = formObj.email;
```

Comment:
I love this implementation ❤️, it is simple and clean 💪🏻!

Reply:
Thank you a lot for the Comment.